### PR TITLE
Update Natural Language

### PR DESCRIPTION
**Pull request by lukesneeringer**
This is an update to the natural language API, both v1 and v1beta2, to add new features. Notably, (e.g. for the release notes), this adds the `analyzeEntitySentiment` method to v1.

@landrito Please sanity check.
@stephenplusplus Once Ernest has sanity checked, please merge and release to npm, and email me when this is done.

### CODE REVIEW

**Comment by lukesneeringer** [comment]
@callmehiphop CI is failing on docs for this (and #2613) and I have no idea why. Could you investigate?

File: packages/language/src/index.js [hunk 1]
```diff
@@ -13,36 +13,46 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-/*!
- * @module language
- * @name Language
```
Comment (callmehiphop) [inline packages/language/src/index.js]: @lukesneeringer just like https://github.com/GoogleCloudPlatform/google-cloud-node/pull/2613#discussion_r139265635 these are required in order to build the docs.

Comment (lukesneeringer) [reply packages/language/src/index.js]: Thank you!

**Comment by lukesneeringer** [comment]
@stephenplusplus Can you release this to npm?

**Comment by stephenplusplus** [comment]
Hey, quick thing I noticed-- we use the `protos` directory in other places in the repo, but the change here uses `proto`. I tried just moving the files to `protos` and updating the references in the `video_intellience_service_client.js` files to match. But I hit an error:

```
Error: ENOENT: no such file or directory, open 'C:\Users\sawch\dev\GCN\packages\video-intelligence\proto\google\cloud\videointelligence\v1beta2\video_intelligence.proto'
    at Object.fs.openSync (fs.js:652:18)
    at Object.fs.readFileSync (fs.js:553:33)
    at fetch (C:\Users\sawch\dev\GCN\packages\video-intelligence\node_modules\protobufjs\src\root.js:160:34)
    at GoogleProtoFilesRoot.load (C:\Users\sawch\dev\GCN\packages\video-intelligence\node_modules\protobufjs\src\root.js:194:13)
    at GoogleProtoFilesRoot.loadSync (C:\Users\sawch\dev\GCN\packages\video-intelligence\node_modules\protobufjs\src\root.js:235:17)
    at Object.loadSync (C:\Users\sawch\dev\GCN\packages\video-intelligence\node_modules\protobufjs\src\index-light.js:69:17)
    at GrpcClient.loadProto (C:\Users\sawch\dev\GCN\packages\video-intelligence\node_modules\google-gax\lib\grpc.js:156:14)
    at Object.<anonymous> (C:\Users\sawch\dev\GCN\packages\video-intelligence\src\index.js:77:31)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\sawch\dev\GCN\packages\video-intelligence\test\gapic-v1beta1.js:19:25)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
    at C:\Users\sawch\dev\GCN\packages\video-intelligence\node_modules\mocha\lib\mocha.js:231:27
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (C:\Users\sawch\dev\GCN\packages\video-intelligence\node_modules\mocha\lib\mocha.js:228:14)
    at Mocha.run (C:\Users\sawch\dev\GCN\packages\video-intelligence\node_modules\mocha\lib\mocha.js:514:10)
    at Object.<anonymous> (C:\Users\sawch\dev\GCN\packages\video-intelligence\node_modules\mocha\bin\_mocha:480:18)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
    at Function.Module.runMain (module.js:609:10)
    at startup (bootstrap_node.js:158:16)
    at bootstrap_node.js:598:3
```

Did I miss a reference, or does this mean something needs to be changed in google-gax?

cc @landrito

**Pull request merged**